include/bat/ads/ad_info.h

```diff
@@ -2,7 +2,8 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-#pragma once
+#ifndef SRC_BRAVE_VENDOR_BAT_NATIVE_ADS_INCLUDE_BAT_ADS_AD_INFO_H_
```

Comment:
these shouldn't have `SRC_BRAVE_VENDOR_` at the start. If you are getting that full path from lint then something is misconfigured with it

Reply:
👍 yes I was getting from lint, so will fix now

Reply:
Fixed